.circleci/config.yml

```diff
@@ -13,18 +13,23 @@ jobs:
       - run:
           name: External Dependencies
           command: |
-            go get github.com/mattn/goveralls
-            go get -u github.com/jstemmer/go-junit-report
-            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-            go get -u github.com/haya14busa/goverage
+            GO111MODULE=off go get github.com/mattn/goveralls
+            GO111MODULE=off go get -u github.com/jstemmer/go-junit-report
+            GO111MODULE=off go get -u github.com/haya14busa/goverage
+            GO111MODULE=off go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
+            git clone git@github.com:Financial-Times/upp-coding-standard.git
       - run:
           name: Test Results
           command: |
             mkdir -p ${CIRCLE_TEST_REPORTS}
             mkdir -p ${CIRCLE_COVERAGE_REPORT}
       - run:
-          name: Go Dep
-          command: dep ensure -vendor-only
+          name: Go Build
+          command: GO111MODULE=on go build -mod=readonly -v
```

Comment:
With Go 1.13 we don't need to set `GO111MODULE=on` even inside GOPATH (given that we have a go.mod file present).